### ISSUE TITLE
Targeting region with dual contours doesn't work

### ISSUE THREAD

**Issue by mcm001** [PhotonVision/photonvision#389]
The target crosshair flicks between top and bottom constantly in dual or 2ormore modes. 

https://user-images.githubusercontent.com/29715865/148980931-21f51683-ca25-492f-a104-83ebbb8589b5.mp4

**Comment by Bankst** [comment]
Is this still an issue?

**Comment by mcm001** [comment]
I don't think so, but we should test again. Iirc i added a unit test for this

**Comment by samfreund** [comment]
Closing as stale, reopen if it's still an issue
